pyiceberg/table/update/spec.py

```diff
@@ -128,6 +133,7 @@ def add_field(
         self._adds.append(new_field)
         return self
 
+
     def add_identity(self, source_column_name: str) -> UpdateSpec:
```

Comment:
I think we need to revert this to make the linter happy:
```suggestion
    def add_identity(self, source_column_name: str) -> UpdateSpec:
```